tests/unit/test_generate_umm_var.py

```diff
@@ -19,6 +19,7 @@ def setUpClass(cls):
         cls.collection_concept_id = 'C1234567890-PROV'
         cls.launchpad_token_header = 'launchpad-token'
         cls.netcdf4_basename = 'f16_ssmis_20210426v7.nc'
+        cls.data_url = 'http://esipfed.org/ns/fedsearch/1.1/data#'
```

Comment:
Sorry, I think you misunderstood what I was trying to say:

For the class var set it up for the full value
``` python
cls.query_granule_return = [
            {
                'links': [
                    {
                        'href': self.netcdf4_url,
                        'rel': 'http://esipfed.org/ns/fedsearch/1.1/data#',
                    }
                ]
            }
        ]
```

And then you can use it everywhere below
``` python
        mock_granule_query.return_value.get.return_value = self.query_granule_return
```

Reply:
ok. makes sense.